docs/src/userguide/special_methods.rst

```diff
@@ -194,6 +194,18 @@ Depending on the application, one way or the other may be better:
 
   These constants can be cimported from the ``cpython.object`` module.
 
+* Use a ``cython.total_ordering`` decorator, which is a re-implementation of
+  the `functools.total_ordering
+  <https://docs.python.org/3/library/functools.html#functools.total_ordering>`_
+  decorator. It can only be used on a ``cdef`` class:
+
+  .. code-block:: cython
+
+    @cython.total_ordering
+    cdef class ExtGe:
+        def __ge__(self, other):
+            return False
```

Comment:
Let's have something a bit more realistic here. It's always good to steer readers in the right direction. These things are not quite so trivial at the C level.
```suggestion
    @cython.total_ordering
    cdef class ExtGe:
        cdef int x

        def __ge__(self, other):
            if not isinstance(other, ExtGe):
                return NotImplemented
            return self.x >= (<ExtGe>other).x

        def __eq__(self, other):
            return isinstance(other, ExtGe) and self.x == (<ExtGe>other).x
```